solr/server/etc/jetty.xml

```diff
@@ -99,6 +100,7 @@
           </New>
         </Arg>
       </Call>
```

Comment:
Do we need a fallback CSP header for all API endpoints?

Reply:
@rmuir You added the CSP stuff to `jetty.xml`. Do you see a reason to keep sending the header for any and all API requests to Solr that are not from a browser? I don't.

@chatman The only other UI I can think of being served by Solr backend are those that are part of a package installed by package manager. Those UIs are 100% stand alone, can we also let it be the UI author's responsibility to secure it with proper headers? I think so.

Reply:
If there's an easy way to provide a "fallback" if the header is not set, we could do that to avoid this discussion. I'm aware of Jetty's generic response rewrite plugins. We also have our `SolrDispatchFilter` but I find no single code point where one could manipulate response header before response is sent.

Reply:
> @rmuir You added the CSP stuff to `jetty.xml`. Do you see a reason to keep sending the header for any and all API requests to Solr that are not from a browser? I don't.

I set these headers across the board for all responses, pure API servers too. For something doing OAuth authorization code flow I would definitely set it.  I don't understand "not from a browser" as this flow usually involves one?

Reply:
Ok, turned out to be much simpler. Just leaving the `HeaderPatternRule` in `jetty.xml` worked fine. The `LoadAdminUiServlet` simply overwrite the header, and everything is just as it has been, no extra code. I pushed the fix, we can close this concern.